storage/src/system/runtime_native.rs

```diff
@@ -479,6 +479,11 @@ where
         &self.runtime_footprint
     }
 
+    /// Returns the addressable entity being used by this instance.
```

Comment:
Just noticed this docu comment is wrong. Should be:

"Returns a mutable reference to the runtime footprint used by this instance."